### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

**Comment by npentrel** [comment]
@jeremyrhyde please review and merge

File: README.md [hunk 1]
```diff
@@ -23,6 +30,9 @@ After creating a new arm component resource and adding this module to your confi
 
 Edit the attributes as applicable.
 
+> [!NOTE]
+> For more information, see [Configure a Machine](https://docs.viam.com/build/configure/).
+
 ## Attributes
 
 The following attributes are available:
```
Comment (sguequierre) [inline README.md:38]: could make required attribute bold

File: README.md [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+
 ## Configure your KUKA Arm
 
-After creating a new arm component resource and adding this module to your config, several attributes can be added to specify certain configurations of your arm:
+> [!NOTE]
+> Before configuring your Kuka Arm, you must [add a machine](https://docs.viam.com/fleet/machines/#add-a-new-machine).
+
+Navigate to the **CONFIGURE** tab of your machine’s page in [the Viam app](https://app.viam.com/). Click the **+** icon next to your machine part in the left-hand menu and select **Component**. Select the `arm` type, then search for and select the `arm / viam-kuka` model. Click **Add module**, then enter a name or use the suggested name for your base and click **Create**.
 
+On the new component panel, copy and paste the following attribute template into your base’s attributes field:
```
Comment (sguequierre) [inline README.md:19]:
```suggestion
On the new component panel, copy and paste the following attribute template into your arm’s attributes field:
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
+> [!NOTE]
+> For more information on modules, see [Modular Resources](https://docs.viam.com/registry/#modular-resources).
+
 ## Configure your KUKA Arm
 
-After creating a new arm component resource and adding this module to your config, several attributes can be added to specify certain configurations of your arm:
+> [!NOTE]
+> Before configuring your Kuka Arm, you must [add a machine](https://docs.viam.com/fleet/machines/#add-a-new-machine).
+
+Navigate to the **CONFIGURE** tab of your machine’s page in [the Viam app](https://app.viam.com/). Click the **+** icon next to your machine part in the left-hand menu and select **Component**. Select the `arm` type, then search for and select the `arm / viam-kuka` model. Click **Add module**, then enter a name or use the suggested name for your base and click **Create**.
```
Comment (sguequierre) [inline README.md:17]:
```suggestion
Navigate to the **CONFIGURE** tab of your machine’s page in [the Viam app](https://app.viam.com/). Click the **+** icon next to your machine part in the left-hand menu and select **Component**. Select the `arm` type, then search for and select the `arm / viam-kuka` model. Click **Add module**, then enter a name or use the suggested name for your arm and click **Create**.
```

File: README.md (hunk 1 above)
Comment (npentrel) [reply README.md:38]: done

**Review by jeremyrhyde — APPROVED** [review]

**Pull request merged**